subtests/docker_cli/deferred_deletion/deferred_deletion.py

```diff
@@ -58,45 +71,36 @@ def initialize(self):
                            "docker daemon command-line options",
                            DockerTestNAError)
 
+        # FIXME: what if future kernels enable by default and remove the knob?
+        sysctl_knob = "/proc/sys/fs/may_detach_mounts"
+        self.failif(not os.path.exists(sysctl_knob),
+                    "sysctl knob %s does not exist; this system is"
+                    " not likely to support deferred deletion" % sysctl_knob,
+                    DockerTestNAError)
```

Comment:
This _could_ happen if ``/proc`` isn't mounted for some reason, so no prob. w/ checking.